.github/workflows/ci-docker-build-publish-image.yml

```diff
@@ -70,7 +75,12 @@ jobs:
         uses: actions/checkout@44c2b7a8a4ea60a981eaca3cf939b5f4305c123b #ping@v4.1.5
 
       - name: Build image
-        run: docker build -t ${{ steps.set-image-name.outputs.image-name }}:${{ steps.set-image-tag.outputs.image-tag }} -f ${{ inputs.application-path }}/Dockerfile .
+        run: |
+          if [ -n "${{ inputs.build-arg }}" ]; then
+            docker build --tag ${{ steps.set-image-name.outputs.image-name }}:${{ steps.set-image-tag.outputs.image-tag }} --file docker/Dockerfile --build-arg GIT_PACKAGE_TOKEN=${{ inputs.build-arg }} .
```

Comment:
litt rart kanskje å ha berre secreten til GIT_PACKAGE_TOKEN inn i build-args, i staden for anten heile verdien eller å berre skruv av/på dette? `GIT_PACKAGE_TOKEN=secret.GITHUB_TOKEN`, men å ta inn heile har vel truleg nokre encoding issues. Uansett hjelper meg vidare dette då. :)

Reply:
Ja, eg som har rota det til... Fiksar no